<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_opcoes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>301</width>
      <height>551</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="botao_inicio_simulacao">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>470</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Iniciar Simulação</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botao_pausar_simulacao">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>510</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Travar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botao_parar_simulacao">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>510</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Parar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botao_avancar_simulacao">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>510</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Avançar</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="tamanho_passo_tela">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Passos/seg :</string>
      </property>
     </widget>
     <widget class="QSlider" name="tamanho_passo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>241</width>
        <height>19</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="pageStep">
       <number>25</number>
      </property>
      <property name="sliderPosition">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>10</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tempo de Simulação(s) : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tamanhoDaSimulacao">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>5</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>3600</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_parametros">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>331</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>261</width>
         <height>269</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Centrais</string>
       </attribute>
       <widget class="QFrame" name="frame_local">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>261</width>
          <height>131</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>5</y>
           <width>91</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Central Local</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_serv_local">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>41</width>
           <height>27</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>20</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_suc_local">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>80</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>60</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_frac_local">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>107</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>20</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Servidores :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Sucesso(%) :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>109</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Fracasso(%) :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_suc_local_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>80</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>50</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>60</y>
           <width>61</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Remoto</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_frac_local_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>107</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>20</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>41</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Local</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>62</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Destino :</string>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>15</y>
           <width>241</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_local_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>135</y>
          <width>261</width>
          <height>131</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>5</y>
           <width>111</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Central Remota</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_serv_remoto">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>41</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>20</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_suc_remoto">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>80</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>50</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_frac_remoto">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>107</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>20</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Servidores :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Sucesso(%) :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Fracasso(%) :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>62</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Destino :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>41</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Local</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>60</y>
           <width>61</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Remoto</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_suc_remoto_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>80</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>50</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_frac_remoto_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>107</y>
           <width>31</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>20</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="Line" name="line_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>15</y>
           <width>241</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>261</width>
         <height>269</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Mensagem</string>
       </attribute>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>261</width>
          <height>251</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_33">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Local-Local :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_34">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>111</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Local-Remoto :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_35">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>111</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Remoto-Local :</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_36">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Remoto-Remoto :</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>55</y>
           <width>41</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>50</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>105</y>
           <width>41</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>25</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>150</y>
           <width>41</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>15</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>200</y>
           <width>41</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>10</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_37">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>60</y>
           <width>21</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_38">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>110</y>
           <width>21</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_39">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>155</y>
           <width>21</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_40">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>205</y>
           <width>21</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_41">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>15</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Tipos de Mensagem :</string>
         </property>
        </widget>
        <widget class="Line" name="line_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>35</y>
           <width>221</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_parametros">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>78</width>
       <height>17</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Parâmetros</string>
     </property>
    </widget>
    <zorder>frame</zorder>
    <zorder>botao_inicio_simulacao</zorder>
    <zorder>botao_pausar_simulacao</zorder>
    <zorder>botao_parar_simulacao</zorder>
    <zorder>botao_avancar_simulacao</zorder>
    <zorder>frame_parametros</zorder>
    <zorder>label_parametros</zorder>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>461</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_logs">
     <attribute name="title">
      <string>Logs</string>
     </attribute>
     <widget class="QTextBrowser" name="caixa_logs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_estatistica">
     <attribute name="title">
      <string>Estatística</string>
     </attribute>
     <widget class="QTextBrowser" name="caixa_estatistica">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>491</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
